constellations/orion/src/domains/family.go

```diff
@@ -0,0 +1,38 @@
+package domains
+
+import (
+	"database/sql"
+	"errors"
+	"regexp"
+	"time"
+)
+
+var TABLE_FAMILY = "families"
+
+//ID,password and primary contact email
+//possibly add family name?
+type Family struct {
+	Id           uint         `json:"id"`
+	CreatedAt    time.Time    `json:"-" db:"created_at"`
+	UpdatedAt    time.Time    `json:"-" db:"updated_at"`
+	DeletedAt    sql.NullTime `json:"-" db:"deleted_at"`
+	PrimaryEmail string       `json:"primaryEmail db:"primary_email"`
+	Password     string       `json:"password" db:"password"`
```

Comment:
Go will automatically transform this to `password` for the database.
However, keep the json part because it does not uncapitalize by default behavior.